### DISCUSSION TITLE
NPM 7 roadmap and the current status

### DISCUSSION THREAD

**Discussion by pubmikeb** [npm/feedback#101] [General]
Currently I see two actively developed versions on https://github.com/npm/cli — v.6.x.x and v.7.x.x.

But there is no any info regarding the actual roadmap for the v.7.
There is a dedicated repo https://github.com/npm/roadmap, but I can't find there planned dates for the GA of the v.7.

Where can I find an updated roadmap for NPM v.7?

**Comment by MylesBorins** [comment]
Hi @pubmikeb 

You can see a view of the roadmap with only CLI related issues [here](https://github.com/npm/roadmap/projects/1?card_filter_query=label%3Acli)

We do have an issue tracking GA [here](https://github.com/npm/roadmap/issues/14). We are targeting the end of this Calendar year, but it could potentially spill in Q1 21'. We are definitely planning to have npm 7 GA in time for Node.js 16.
